spec.bs

```diff
@@ -3683,87 +3705,34 @@ The following step will be added to the [=HTTP-network-or-cache fetch=] algorith
 
 </div>
 
-<div algorithm="fetch per traversable navigable structures patch">
-Modify [[FETCH]]'s [[FETCH#infrastructure]] to add a new section called "Per Traversable Navigable
-Structures", with the following content:
-
-<h3 id=direct-from-sellers-signals-key-struct>Direct from seller signals key</h3>
-A <dfn>direct from seller signals key</dfn> is a [=struct=] with the following [=struct/items=]:
-
-NOTE: This is only captured during a [=request=] whose [=request/initiator type=] is `"fetch"`, made
-with the {{RequestInit/adAuctionHeaders}} option set to `true`, as described in the
-[:Ad-Auction-Signals:] header description.
-
-<dl dfn-for="direct from seller signals key">
-: <dfn>seller</dfn>
-:: An [=origin=]. Matches the origin that served the captured [:Ad-Auction-Signals:] header.
-: <dfn>ad slot</dfn>
-:: A [=string=]. Matches the `adSlot` key of the JSON dictionaries in the top-level array of the
-  [:Ad-Auction-Signals:] value.
-
-</dl>
-
-<h3 id=direct-from-sellers-signals-struct>Direct from seller signals</h3>
-A <dfn>direct from seller signals</dfn> is a [=struct=] with the following [=struct/items=]:
-
-NOTE: This is only captured during a [=request=] whose [=request/initiator type=] is `"fetch"`, made

```

Comment:
ditto

Reply:
it was moved to [here](https://github.com/WICG/turtledove/pull/888/files#diff-6f5a1d8263b0b0c42e2716ba5750e3652e359532647ac934c1c70086ae3ceddaR3762).